### ISSUE TITLE
[Feature request] Reverse rotation

### ISSUE THREAD

**Issue by parkerlreed** [joukos/PaperTTY#59]
`stdin` for example has `--portrait` but nothing for flipping the display upside down.

**Comment by joukos** [comment]
Yeah, there's some inconsistency there currently (and `stdin` hasn't been touched for a long time). The orientation settings should be part of the driver's features instead of having varying options for the subcommands. Doing it (in a simple, not necessarily the best way) should be fairly easy, but cleaning up afterwards and testing that nothing breaks might take a while longer. I don't know when I'll have the time for it.
